packages/react-native/src/Authenticator/Defaults/VerifyUser/VerifyUser.tsx

```diff
@@ -82,12 +82,12 @@ const FormFields: DefaultVerifyUserComponent['FormFields'] = ({
 }) => {
   return (
     <RadioGroup disabled={isPending}>
-      {fields.map(({ name, value, ...props }) => (
+      {fields.map(({ label, name, value, ...props }) => (
         <Radio
           {...props}
-          key={value}
-          value={value}
-          label={censorContactInformation(name, value)}
+          key={name}
+          value={name}
+          label={censorContactInformation(label, value)}
```

Comment:
Wouldn't `name` still make more sense to use as parameter here? Since customer might change `label` to some custom string.

Reply:
Thanks for catching this! Addressed.